HeterogeneousCore/AlpakaUtilities/interface/demangle.h

```diff
@@ -0,0 +1,13 @@
+#ifndef Framework_demangle_h
+#define Framework_demangle_h
+
+#include <boost/core/demangle.hpp>
+
+namespace edm {
```

Comment:
Put the inline variable to some other namespace than `edm`. We have also other demanglers within CMSSW, e.g.
https://github.com/cms-sw/cmssw/blob/107a5fed908622757511062f46961c0f7552fba8/FWCore/Utilities/interface/TypeDemangler.h#L6-L7
(although that particular function is geared to be compatible with the convention of reflex)

Reply:
Apparently this is not used (at least at the moment), so I've removed it.